happex/main.py

```diff
@@ -1,16 +1,58 @@
 import json
 from flask import render_template
+import plotly.offline as py
+import plotly.graph_objs as go
 from happex import app
 
-# import matplotlib.pyplot as plt
-
 
 @app.route("/")
 def index():
     """Homepage - worldmap data or options for each page?"""
-    return json.dumps(list(map(lambda x: (x[0], float(x[1])), app.db.get_scores())))
-
-
+    return json.dumps(list(map(lambda x: (x[0], float(x[1])), app.db.get_all_scores())))
+
+def worldgraph1(ctry, data, ccode, title='Worldwide Happiness', reverse=False):
+    fig = go.Figure(data=go.Choropleth(
+        locations=ccode,
+        z=data,
+        text=ctry,
+        autocolorscale=True,
+        reversescale=reverse,
+        marker_line_color='darkgray',
+        marker_line_width=0.5,
+        colorbar_title='Happiness',))
+    fig.update_layout(
+        title_text=title,
+        geo=dict(
+            showframe=False,
+            showcoastlines=False,
+            projection_type='equirectangular'),
+        annotations=[dict(
+            x=0.55,
+            y=0.1,
+            xref='paper',
+            yref='paper',
+            text='Source: Worldwide Happiness Report, 2019',
+            showarrow=False)])
+    fig.show()
+
+def worldgraph2(ctry, data, ccode, title='Worldwide Happiness', reverse=False):
```

Comment:
Non-route functions in main can be moved to another module (plot or graph?) for clarity.

Reply:
Functions moved to its own module